tests/full/test.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+USER="tmt-tester"
+USER_HOME="/home/$USER"
+
+CONNECT_RUN="/tmp/CONNECT"
+
+BRANCH="${BRANCH:-}"
+# Set following to 1 if you are running pre-relase
+PRE_RELEASE="${PRE_RELEASE:-0}"
+
+# Set to `test <options>` for test filtering
+TEST_CMD="${TEST_CMD:-}"
+
+set -o pipefail
+
+rlJournalStart
+    rlPhaseStartSetup
+
+        if [[ $PRE_RELEASE -eq 1 ]]; then
+            [[ -z "$BRANCH" ]] && rlDie "Please set BRANCH when running pre-release"
+        fi
+
+        if ! rlIsFedora; then
+          rlRun "rlImport epel/epel"
+          rlRun "dnf config-manager --set-enabled epel"
+
+          for repo in powertools extras crb codeready-builder; do
+            real_repo_name="$(dnf repolist --all | grep -Eo "[-a-zA-Z0-9_]*$repo[-a-zA-Z0-9_]*" | head -n1)"
+            if [[ -n "$real_repo_name" ]]; then
+               rlRun "dnf config-manager --set-enabled $real_repo_name"
+            fi
+          done
+        fi
+
+        rlFileBackup /etc/sudoers
+        id $USER &>/dev/null && {
+            rlRun "pkill -9 -u $USER" 0,1
+            rlRun "loginctl terminate-user $USER" 0,1
+            rlRun "userdel -r $USER" 0 "Removing existing user"
+        }
+        rlRun "useradd $USER"
+        rlRun "echo '$USER ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers" 0 "password-less sudo for test user"
+        rlRun "chmod 400 /etc/sudoers"
+        rlRun "loginctl enable-linger $USER" # start session so /run/ directory is initialized
+
+        # Making sure USER can r/w to the /var/tmp/tmt
+        test -d /var/tmp/tmt && rlRun "chown $USER:$USER /var/tmp/tmt"
+
+        # Clone repo
+        rlRun "git clone https://github.com/teemtee/tmt $USER_HOME/tmt"
+        rlRun "pushd $USER_HOME/tmt"
+        [ -n "$BRANCH" ] && rlRun "git checkout --force '$BRANCH'"
+        # Make current commit visible in the log
+        rlRun "git show -s | cat"
+        # Do not "patch" version for pre-release...
+        [[ $PRE_RELEASE -ne 1 ]] && rlRun "sed 's/^Version:.*/Version: 9.9.9/' -i tmt.spec"
+
+        # Build tmt packages
+        rlRun "dnf builddep -y tmt.spec" 0 "Install build dependencies"
+        rlRun "make rpm" || rlDie "Failed to build tmt rpms"
+
```

Comment:
We need to add install of freshly build tmt packages here. Until this moment test does not depend on tmt binary so it can be used as bootstrapping (trying to build tmt on new centos/rhel)

Reply:
I see, thanks for clarification and the fix.